azkaban-common/src/test/java/azkaban/trigger/BasicTimeCheckerTest.java

```diff
@@ -102,4 +106,55 @@ public void cronTimerTest1() {
     assertTrue(cond.getNextCheckTime() == year2050.getMillis());
   }
 
+  /**
+   * Test when PST-->PDT happens in 2020.
```

Comment:
Add links to an article that describes how the transition works?
Describe how our system works?